src/arm/BB-BBGG-WL1835-00A0.dts

```diff
@@ -37,6 +37,14 @@
 				gpio = <&gpio3 9 0>;
 				enable-active-high;
 			};
+			leds {
+				led6 {
+					label = "beaglebone:green:Wlan";
```

Comment:
On BeagleBone AI, we call this LED "usr4" and configure it to show network connection and activity. I'd recommend we do the same with this LED.